app/src/main/java/org/coepi/android/ui/alertsdetails/AlertsDetailsViewModel.kt

```diff
@@ -16,15 +18,30 @@ class AlertsDetailsViewModel(
     val report: LiveData<List<AlertDetailsSymptomViewData>> = just(args.report.toViewData())
         .toLiveData()
 
-    // FIXME temporary title to identify the report. No design yet for this screen.
-    val title: LiveData<String> = just("Report: ${args.report.id}")
+    val title: LiveData<String> = just(toMonthAndDay(args.report.contactTime))
         .toLiveData()
 
+    val exposureTime: LiveData<String> = just(toTime(args.report.contactTime))
+        .toLiveData()
+
+    //TODO Show when the symptoms were reported. Maybe the earliestSymptom Date?
+    val reportedTime: LiveData<String> = just(" ").toLiveData()
```

Comment:
It's better to not add incomplete implementations. If really necessary, add a TODO with a short explanation of why it's not implemented yet.